go/systests/multiuser_common_test.go

```diff
@@ -190,7 +190,6 @@ func (smc *smuContext) setupDeviceHelper(u *smuUser, puk bool) *smuDeviceWrapper
 	tctx := setupTest(smc.t, u.usernamePrefix)
 	tctx.Tp.DisableUpgradePerUserKey = !puk
 	tctx.G.SetClock(smc.fakeClock)
-	installInsecureTriplesec(tctx.G)
```

Comment:
Removed extra call, it's already set up by `setupTest` (called 3 lines above this one) here: https://github.com/keybase/client/blob/master/go/systests/common_test.go#L22